jaas/src/main/java/com/griddynamics/jagger/jaas/config/ApplicationConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.griddynamics.jagger.jaas.config;
+
+import com.griddynamics.jagger.dbapi.DatabaseService;
+import com.griddynamics.jagger.engine.e1.services.DataService;
+import com.griddynamics.jagger.engine.e1.services.DefaultDataService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.*;
+
+@PropertySources({@PropertySource("classpath:/basic/default.environment.properties")})
+@ImportResource({"classpath:/common/storage.rdb.client.conf.xml", "classpath:/spring/dbapi.config.xml"})
+@Configuration
+public class ApplicationConfig {
```

Comment:
Many configuration files don't separate root context - all beans will be in one application context after bootstrap. It just makes your configuration clear and easier to search beans, since they all will be grouped. Moreover, if you use spring profiles it's much easier with separated config classes, since all you need is just add annotation to class, otherwise you'll need to add these annotations to every bean.